modules/core/src/main/scala/lucuma/odb/api/model/ConstraintSetModel.scala

```diff
@@ -5,26 +5,23 @@ package lucuma.odb.api.model
 
 import cats._
 import cats.data.State
-import cats.mtl.Stateful
 import cats.syntax.all._
 import clue.data.Input
 import eu.timepit.refined.types.string.NonEmptyString
 import io.circe.Decoder
 import io.circe.generic.semiauto._
 import io.circe.refined._
 import lucuma.core.enum._
-import lucuma.core.model.{ConstraintSet, Program}
+import lucuma.core.model.Observation
 import lucuma.core.optics.syntax.lens._
-import lucuma.odb.api.model.Existence._
 import lucuma.odb.api.model.syntax.input._
 import monocle.{Fold, Lens, Optional}
 import monocle.macros.GenLens
 
+import scala.collection.immutable.SortedSet
+
 final case class ConstraintSetModel(
-  id:              ConstraintSet.Id,
-  existence:       Existence,
-  programId:       Program.Id,
-  name:            NonEmptyString,
+  name:            NonEmptyString,  // maybe we eliminate this?
```

Comment:
Andy makes a good point about otherwise identical constraint sets that would be distinguished only on timing windows (e.g., Io vs Europa).  We could make the name optional though and perhaps most users would ignore it if we can default something based on CC, IQ, etc.